.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,45 @@
+name: Benchmarks
+
+on: [push, pull_request]
+
+jobs:
+  ubuntu-build:
+    name: Build - Ubuntu
+
+    strategy:
+      matrix:
+        os: ['ubuntu-22.04']
+        build_type: [Release]
+        compiler: [{c: gcc, cxx: g++}]
+    runs-on: ${{matrix.os}}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Install apt packages
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y cmake libnuma-dev
+
+      - name: Configure build
+        run: >
+          cmake
+          -B ${{github.workspace}}/build
+          -DCMAKE_BUILD_TYPE=${{matrix.build_type}}
+          -DCMAKE_C_COMPILER=${{matrix.compiler.c}}
+          -DCMAKE_CXX_COMPILER=${{matrix.compiler.cxx}}
+          -DUMF_BUILD_BENCHMARKS=ON
+          -DUMF_BUILD_TESTS=OFF
+          -DUMF_BUILD_LIBUMF_POOL_JEMALLOC=OFF
+          -DUMF_FORMAT_CODE_STYLE=OFF
+          -DUMF_DEVELOPER_MODE=OFF
+          -DUMF_BUILD_LIBUMF_POOL_JEMALLOC=OFF
```

Comment:
You set UMF_BUILD_LIBUMF_POOL_JEMALLOC twice. BTW - shouldn't we add jemalloc to benchmark also?

Reply:
Yes, but in a separate PR.
Done.